action-server/covidflow/utils/geocoding.py

```diff
@@ -0,0 +1,40 @@
+import os
+from typing import Optional, Tuple
+
+import googlemaps
+import structlog
+
+logger = structlog.get_logger()
+
+GOOGLE_API_KEY_ENV = "GOOGLE_GEOCODING_API_KEY"
+
+GEOMETRY = "geometry"
+LOCATION = "location"
+LATITUDE = "lat"
+LONGITUDE = "lng"
+
+
+class Geocoding:
+    def __init__(self):
+        key = os.environ.get(GOOGLE_API_KEY_ENV)
```

Comment:
Why not use `os.environ[GOOGLE_API_KEY_ENV]` if you raise when it's `None` anyways?

Reply:
right.